cartography/intel/lastpass/users.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+from dateutil import parser as dt_parse
+from requests import Session
+
+from cartography.client.core.tx import load_graph_data
+from cartography.graph.querybuilder import build_ingestion_query
+from cartography.models.lastpass.tenant import LastpassTenantSchema
+from cartography.models.lastpass.user import LastpassUserSchema
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+# Connect and read timeouts of 60 seconds each; see https://requests.readthedocs.io/en/master/user/advanced/#timeouts
+_TIMEOUT = (60, 60)
+
+
+@timeit
+def sync(
+    neo4j_session: neo4j.Session,
+    lastpass_provhash: str,
+    common_job_parameters: Dict[str, Any],
```

Comment:
I think we should include `update_tag` and `tenant_id` as separate parameters to `sync` in addition to including them as keys in `common_job_parameters`. The AWS modules do similar.

Reply:
Hum ... I can make the change but do not realy like that, seems very confusing for me :
https://github.com/lyft/cartography/blob/672b5fe4918fef562d7816d8e9833269c801156f/cartography/intel/aws/emr.py#L96

Here you use update_tag for load function and common_parameters for cleanup.
IMHO if for any reason in the code those two parameters become different you will have a uggly cleanup very hard to debug

Reply:
I agree that this is a problem but all the other syncs currently follow that pattern, and update_tag is never written to (fingers crossed that it stays this way).

In future I would like to do a larger refactor to eliminate the possibility of those two parameters becoming different. This could involve:

(A) make `common_job_parameters` become a formal object where `update_tag` and `tenant_id` can become their own fields 

or

(B) Get rid of `common_job_parameters` altogether and just use parameters from the function itself

or

(C) something else


However for right now I think this module should keep the existing pattern.